### PR TITLE
Add support for `spending_limits` on Issuing Card and Cardholder

### PR DESCRIPTION
**Pull request by remi-stripe**
Related to https://github.com/stripe/stripe-go/pull/831

r? @ob-stripe 
cc @stripe/api-libraries

### CODE REVIEW

**Comment by remi-stripe** [comment]
cc @zacht-stripe to triple check the changes and make sure I did not miss anything

File: src/Stripe.net/Entities/Issuing/Cards/AuthorizationControls.cs [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
         [JsonProperty("blocked_categories")]
         public List<string> BlockedCategories { get; set; }
 
+        /// <summary>
+        /// Limit the spending with rules based on time intervals and categories.
+        /// </summary>
+        [JsonProperty("spending_limits")]
+        public SpendingLimits SpendingLimits { get; set; }
+
+        [Obsolete("This is now unsupported")]
```
[the remark this answers is not in the document]
Comment (remi-stripe) [reply src/Stripe.net/Entities/Issuing/Cards/AuthorizationControls.cs]: Hmmm I was not sure what to put. Like there's no replacement or anything. It's just removed as a property.

[the remark this answers is not in the document]
Comment (remi-stripe) [reply src/Stripe.net/Entities/Issuing/Cards/AuthorizationControls.cs]: @ob-stripe What do you think of this one?

[the remark this answers is not in the document]
Comment (ob-stripe) [reply src/Stripe.net/Entities/Issuing/Cards/AuthorizationControls.cs]: Did you mean to change it? The message hasn't changed.

[the remark this answers is not in the document]
Comment (remi-stripe) [reply src/Stripe.net/Entities/Issuing/Cards/AuthorizationControls.cs]: I am asking you what I should put. This option is just entirely removed, it's not replaced by anything. So I don't know what to put beyond "please stop using it". Do you have any idea what string would be enough for you?

**Comment by remi-stripe** [comment]
Turns out `spending_limits` is an array, not a hash. So I've pushed a new version of it. @ob-stripe should we rename `SpendingLimits` to `SpendingLimit` instead so that you have `List<SpendingLimit> SpendingLimits;`? I'm assuming yes

**Comment by ob-stripe** [comment]
> should we rename `SpendingLimits` to `SpendingLimit` instead so that you have `List<SpendingLimit> SpendingLimits;`? I'm assuming yes

Yes please.

**Comment by remi-stripe** [comment]
@ob-stripe okay fixed. Can I ask you to re-review carefully? I broke stripe-go so want to make ~sure it's correct :p

**Comment by the800kid** [comment]
Hi @ob-stripe / @remi-stripe 
Just wondering if these changes are going to make it into an official release?  I am in need of this feature.  Thanks.

**Comment by remi-stripe** [comment]
@the800kid Thanks for reaching out! I paused those changes for now as we're changing the shape of the API a little bit so I was waiting for someone to need this before releasing it.

Since you need it though, I'll work on finishing those today!

**Comment by remi-stripe** [comment]
Closing in favour of https://github.com/stripe/stripe-dotnet/pull/1877

**Comment by the800kid** [comment]
@remi-stripe Thank you for the quick reply and action.   I see you have made changes to the models since the original work.  Looking forward to being able to use the feature.  Thanks again!

**Comment by remi-stripe** [comment]
@the800kid This was released in 34.4.0! Let me know if you encounter any issues!

**Pull request closed without merging**